components/debug/debugging.hpp

```diff
@@ -9,6 +9,9 @@
 #include <SDL_messagebox.h>
 
 #include "debuglog.hpp"
+#if defined _WIN32
+#   include "win32.hpp"
```

Comment:
This should probably be included in the `.cpp` file instead to reduce compile times for things that include this file. It doesn't make much difference here, as the header is tiny, but it's best to stick to best practices even when it doesn't matter so you stay in the habit.

Reply:
Would you rather have me move it to the central debug file or rename "win32" to, say, "win32console"?

Reply:
At risk of sounding a bit like KenM's ["I am only certified to diagnose item deficiency"](https://i.imgur.com/0C4JHZR.jpg), I'm not sure. Someone else might have a preference.

Reply:
IMO it makes sense to move win32 into the central debug file, since it's a mere init method that won't be used by anything else.